eng/versioning.targets

```diff
@@ -29,6 +29,10 @@
     </AssemblyAttribute>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetsBrowser)' == 'true' and '$(IsTestProject)' != 'true'">
```

Comment:
What about libraries that are platform-neutral and `'$(TargetsAnyOS)`' == 'true'`? I tried changing this to the following to see what the results were:

```xml
<ItemGroup Condition="'$(IsTestProject)' != 'true' and ('$(TargetsBrowser)' == 'true' or '$(TargetsAnyOS)' == 'true')">
```

With that, I see 56 additional errors across several libraries.

```txt
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.ComponentModel.TypeConverter\src\System\ComponentModel\PropertyDescriptor.cs(192,24): error CA1416: 'TypeDescriptor.CreateInstance(IServiceProvider, Type, Type[], object[])' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.ComponentModel.TypeConverter\src\System.ComponentModel.TypeConverter.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.ComponentModel.TypeConverter\src\System\ComponentModel\DelegatingTypeDescriptionProvider.cs(36,20): error CA1416: 'TypeDescriptionProvider.CreateInstance(IServiceProvider, Type, Type[], object[])' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.ComponentModel.TypeConverter\src\System.ComponentModel.TypeConverter.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.ComponentModel.TypeConverter\src\System\ComponentModel\PropertyDescriptor.cs(195,20): error CA1416: 'TypeDescriptor.CreateInstance(IServiceProvider, Type, Type[], object[])' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.ComponentModel.TypeConverter\src\System.ComponentModel.TypeConverter.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.ComponentModel.TypeConverter\src\System\ComponentModel\TypeDescriptor.cs(3074,24): error CA1416: 'TypeDescriptionProvider.CreateInstance(IServiceProvider, Type, Type[], object[])' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.ComponentModel.TypeConverter\src\System.ComponentModel.TypeConverter.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Diagnostics.TextWriterTraceListener\src\System\Diagnostics\XmlWriterTraceListener.cs(263,45): error CA1416: 'Process.GetCurrentProcess()' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Diagnostics.TextWriterTraceListener\src\System.Diagnostics.TextWriterTraceListener.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Diagnostics.TextWriterTraceListener\src\System\Diagnostics\XmlWriterTraceListener.cs(264,51): error CA1416: 'Process.ProcessName.get' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Diagnostics.TextWriterTraceListener\src\System.Diagnostics.TextWriterTraceListener.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Linq.Parallel\src\System\Linq\Parallel\Scheduling\OrderPreservingPipeliningSpoolingTask.cs(134,25): error CA1416: 'Monitor.Wait(object)' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Linq.Parallel\src\System.Linq.Parallel.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Linq.Parallel\src\System\Linq\Parallel\Merging\OrderPreservingPipeliningMergeHelper.cs(402,25): error CA1416: 'Monitor.Wait(object)' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Linq.Parallel\src\System.Linq.Parallel.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.ServicePoint\src\System\Net\ServicePointManager.cs(105,68): error CA1416: 'EncryptionPolicy.RequireEncryption' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.ServicePoint\src\System.Net.ServicePoint.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebProxy\src\System\Net\WebProxy.cs(185,59): error CA1416: 'Dns.GetHostName()' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebProxy\src\System.Net.WebProxy.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebProxy\src\System\Net\WebProxy.cs(185,42): error CA1416: 'Dns.GetHostEntry(string)' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebProxy\src\System.Net.WebProxy.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebProxy\src\System\Net\WebProxy.cs(185,42): error CA1416: 'IPHostEntry.AddressList.get' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebProxy\src\System.Net.WebProxy.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebProxy\src\System\Net\WebProxy.cs(174,34): error CA1416: 'IPGlobalProperties.GetIPGlobalProperties()' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebProxy\src\System.Net.WebProxy.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebProxy\src\System\Net\WebProxy.cs(174,34): error CA1416: 'IPGlobalProperties.DomainName.get' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebProxy\src\System.Net.WebProxy.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(32,103): error CA1416: 'ClientWebSocketOptions.Proxy.set' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(98,21): error CA1416: 'SocketsHttpHandler.UseCookies.set' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(123,25): error CA1416: 'SocketsHttpHandler.SslOptions.get' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(110,25): error CA1416: 'ClientWebSocketOptions.Proxy.get' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(97,47): error CA1416: 'ClientWebSocketOptions.Cookies.get' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(122,65): error CA1416: 'X509Certificate2Collection' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(121,38): error CA1416: 'SslClientAuthenticationOptions.ClientCertificates.get' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(73,21): error CA1416: 'ClientWebSocketOptions.RemoteCertificateValidationCallback.get' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(122,25): error CA1416: 'SocketsHttpHandler.SslOptions.get' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(70,22): error CA1416: 'ClientWebSocketOptions.UseDefaultCredentials.get' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(69,21): error CA1416: 'ClientWebSocketOptions.Credentials.get' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(121,38): error CA1416: 'SocketsHttpHandler.SslOptions.get' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(107,47): error CA1416: 'ClientWebSocketOptions.Credentials.get' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(99,21): error CA1416: 'SocketsHttpHandler.SslOptions.get' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(99,78): error CA1416: 'ClientWebSocketOptions.RemoteCertificateValidationCallback.get' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(116,41): error CA1416: 'ClientWebSocketOptions.Proxy.get' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(122,25): error CA1416: 'SslClientAuthenticationOptions.ClientCertificates.set' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(107,25): error CA1416: 'SocketsHttpHandler.Credentials.set' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(101,25): error CA1416: 'ClientWebSocketOptions.UseDefaultCredentials.get' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(84,29): error CA1416: 'SocketsHttpHandler.UseCookies.set' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(98,42): error CA1416: 'ClientWebSocketOptions.Cookies.get' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(80,35): error CA1416: 'SocketsHttpHandler' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(97,21): error CA1416: 'SocketsHttpHandler.CookieContainer.set' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(191,21): error CA1416: 'ClientWebSocketOptions.KeepAliveInterval.get' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(112,25): error CA1416: 'SocketsHttpHandler.UseProxy.set' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(116,25): error CA1416: 'SocketsHttpHandler.Proxy.set' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(96,21): error CA1416: 'SocketsHttpHandler.PooledConnectionLifetime.set' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(72,21): error CA1416: 'ClientWebSocketOptions.Cookies.get' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(71,21): error CA1416: 'ClientWebSocketOptions.Proxy.get' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(123,25): error CA1416: 'SslClientAuthenticationOptions.ClientCertificates.get' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(99,21): error CA1416: 'SslClientAuthenticationOptions.RemoteCertificateValidationCallback.set' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(123,72): error CA1416: 'ClientWebSocketOptions.ClientCertificates.get' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(103,25): error CA1416: 'SocketsHttpHandler.Credentials.set' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(95,31): error CA1416: 'SocketsHttpHandler' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(123,25): error CA1416: 'X509CertificateCollection.AddRange(X509CertificateCollection)' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(83,29): error CA1416: 'SocketsHttpHandler.UseProxy.set' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(114,30): error CA1416: 'ClientWebSocketOptions.Proxy.get' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System\Net\WebSockets\WebSocketHandle.Managed.cs(82,29): error CA1416: 'SocketsHttpHandler.PooledConnectionLifetime.set' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Net.WebSockets.Client\src\System.Net.WebSockets.Client.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Private.Xml\src\System\Xml\XmlDownloadManagerAsync.cs(38,21): error CA1416: 'HttpClientHandler.Proxy.set' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Private.Xml\src\System.Private.Xml.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Private.Xml\src\System\Xml\XmlDownloadManagerAsync.cs(34,21): error CA1416: 'HttpClientHandler.Credentials.set' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Private.Xml\src\System.Private.Xml.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Transactions.Local\src\System\Transactions\CommittableTransaction.cs(111,26): error CA1416: 'Monitor.Wait(object)' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Transactions.Local\src\System.Transactions.Local.csproj]
C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Transactions.Local\src\System\Transactions\CommittableTransaction.cs(181,26): error CA1416: 'Monitor.Wait(object)' is unsupported on 'browser' [C:\Users\jeffhand\git\dotnet\runtime\src\libraries\System.Transactions.Local\src\System.Transactions.Local.csproj]
```

@stephentoub, @lewing, @MaximLipnin -- Should we expand this to also warn on platform-neutral libraries and resolve those warnings, or should that be handled separately?

Reply:
By `platform-neutral` you mean multi-targeted right? I don't think `'$(TargetsAnyOS)' == 'true'` means multi-targeted. Anyways I don't we want to see browser warning for any OS

Reply:
I'm referring to libraries that target `$(NetCoreAppCurrent)` without a target framework suffix at all. [For those](https://github.com/dotnet/runtime/blob/master/eng/targetframeworksuffix.props#L85-L89), we set `TargetsAnyOS` to `true`. As far as I know, we'd expect those libraries work in browser (excluding any APIs marked as unsupported), but `TargetsBrowser` will be `false`.

Reply:
> Should we expand this to also warn on platform-neutral libraries and resolve those warnings

We don't need to block this PR on this but we **have to** resolve these warnings as they are real problems which customers could in many cases easily hit

Reply:
> With that, I see 56 additional errors across several libraries.

I see 70 warnings, seems @jeffhandley's build haven't updated the analyzer version. 38 of which related to `System.Net.WebSockets.Client.WebSocketHandle.Managed.cs` which is only built when `'$(TargetsBrowser)' != 'true'`. I have tried to add this to the MBbuild condition but it is not working, MBbuild  might not support so many conditional combinations, I will dig it more

Other warnings look valid and i tried to fix them with my latest commit, please review